core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -1671,6 +1671,83 @@ public void enableDeduplicationWithUpsertKeys(LongList columnsIndexes) {
         }
     }
 
+    @Override
+    public void forceRemovePartitions(LongList partitions) {
```

Comment:
can we have more descriptive parameter name. More appropriate would be `partitionTimestamps`

Reply:
done